examples/index.html

```diff
@@ -9,31 +9,37 @@
     <link rel="stylesheet" href="../node_modules/leaflet/dist/leaflet.css">
     <link rel="stylesheet" href="../dist/vendor.css">
     <link rel="stylesheet" href="../dist/leaflet.distortableimage.css" media="screen" title="no title">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet-control-geocoder/dist/Control.Geocoder.css" />
 
     <script src="../node_modules/leaflet/dist/leaflet.js" type="text/javascript" charset="utf-8"></script>
     <script src="../dist/vendor.js"></script>
     <script src="../dist/leaflet.distortableimage.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=<key>&libraries=places"></script>
+    <script src="https://unpkg.com/leaflet-control-geocoder/dist/Control.Geocoder.js"></script>
   </head>
-  <body style="margin:0;">
-    <form id="test_form">
-      <input type="file" class="ldi" id="inputimage" accept="image/*" />
-    </form>
+<body style="margin:0;">
+  <form id="test_form">
+    <input type="file" class="ldi" id="inputimage" accept="image/*" />
+  </form>
 
-    <div id="map" style="width:100%; height:100%; position:absolute; top:0;"></div>
-  </body>
-  <script>
-    let map;
+  <div id="map" style="width:100%; height:100%; position:absolute; top:0;"></div>
+</body>
+<script>
+  let map;
 
-    (function() {
-      map = L.map('map').setView([51.505, -0.09], 13);
-      map.addGoogleMutant();
+  (function() {
+    map = L.map('map').setView([51.505, -0.09], 13);
+    map.addGoogleMutant();
 
-      map.whenReady(function() {
-        img = L.distortableImageOverlay('example.jpg', {
-          selected: true,
-          fullResolutionSrc: 'large.jpg',
-        }).addTo(map);
-      });
-    })();
-  </script>
-</html>
+    map.whenReady(function() {
+      img = L.distortableImageOverlay('example.jpg', {
+        selected: true,
+        fullResolutionSrc: 'large.jpg',
+      }).addTo(map);
+    });
+  })();
+  
+  L.Control.geocoder().addTo(map);
```

Comment:
You're almost there! There are still some indentation changes showing: 

<img width="970" alt="image" src="https://user-images.githubusercontent.com/24359/196001216-ee1132f3-74c9-431a-9c27-4bf156279dcd.png">

However this line (42) is the one you want, right?

Reply:
12, 18, and 42

Reply:
https://github.com/publiclab/Leaflet.DistortableImage/pull/1158/commits/77c4bf1c3c6a161abb167817b4cc94f3fb0257b1
please review this pr, I have removed all the indentation here. Thanks :)

Reply:
Hi, it looks like there is still some difference in the final diff:

<img width="750" alt="image" src="https://user-images.githubusercontent.com/24359/196001633-40c5bf40-64a8-4a52-bbbb-d18865eb3aae.png">

What I do to check is look on this tab above: https://github.com/publiclab/Leaflet.DistortableImage/pull/1158/files

That is the combined effect of all commits on this PR. Thanks for your work on this, you're almost there!